script.js

```diff
@@ -2,6 +2,7 @@ const BASE_URL = "https://guya.moe";
 const FS_URL = "https://guya.moe/fs";
 const MB_URL = "https://guya.moe/mb";
 const HT_URL = "https://guya.moe/ht";
+const PB_URL = "https://guya.moe/pb";
```

Comment:
Looks like you missed one :p

Reply:
Ah. I did ctrl+f  'pastebin'. Just a quick fix

Reply:
And done